cypress/locators/step-sequence/locators.js

```diff
@@ -0,0 +1,5 @@
+// component preview locators
+export const STEP_SEQUENCE = '#story-root > ol';
```

Comment:
should we start from story_root ?

If so, maybe we should import storyRoot() ? as U used below

Reply:
I isolate `#story-root` to local const becasue is not recommended to share locators between components